packages/chart/src/Scatter.ts

```diff
@@ -103,16 +103,27 @@ export class Scatter extends XYAxis {
                 default:
             }
         }
-
+        const usePointSizeColumn = this.pointSizeColumn() !== null;
+        let pointSizeColumnIdx;
+        let pointMinValue;
+        let pointMaxValue;
+        if (usePointSizeColumn) {
+            pointSizeColumnIdx = this.columns().indexOf(this.pointSizeColumn());
+            pointMinValue = Math.min.apply(this, this.data().map(n => n[pointSizeColumnIdx]));
+            pointMaxValue = Math.max.apply(this, this.data().map(n => n[pointSizeColumnIdx]));
+        }
         const layerColumns = this.layerColumns(host);
         const layerData = this.layerData(host);
         const flatData = this.flattenData(layerColumns, layerData).map(function (d) {
             d.shape = mapShape(context.pointShape());
             d.column = layerColumns[d.colIdx];
             d.row = layerData[d.rowIdx];
-            return d;
-        });
 
+            const valRatio = (d.row[pointSizeColumnIdx] - pointMinValue) / (pointMaxValue - pointMinValue);
+            d.size = usePointSizeColumn ? context.minPointSize() + (valRatio * (context.maxPointSize() - context.minPointSize())) : context.pointSize();
```

Comment:
I thought you were going to use d3-scale for this (you can set the domain to be the "extent" and the range to be min/max point size...